Sources/Bot/Extensions/URL+QueryItems.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension URL {
+    func addingQueryItems(_ queryItems: [URLQueryItem]) -> URL {
+        guard queryItems.isEmpty == false else { return self }
+        
+        var components = URLComponents(url: self, resolvingAgainstBaseURL: true)!
+        components.queryItems = components.queryItems.map { $0 + queryItems } ?? queryItems
```

Comment:
I find this to be more readable, but in the end it's just a matter of taste, so... up to you.
```suggestion
        components.queryItems = (components.queryItems ?? []) + queryItems
```